docm/GATK_haplotype_caller.cwl

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "HaplotypeCaller (GATK 3.6)"
+baseCommand: ["/usr/bin/java", "-jar", "/opt/GenomeAnalysisTK.jar", "-T", "HaplotypeCaller"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 8000
+arguments:
+    ["-gt_mode", "GENOTYPE_GIVEN_ALLELES",
+    "-o", { valueFrom: $(runtime.outdir)/docm_out.vcf }]
+inputs:
+    reference:
+        type: File
+        inputBinding:
+            prefix: "-R"
+            position: 1
+        secondaryFiles: [.fai, ^.dict]
+    normal_bam:
```

Comment:
Do we want to genotype the normal or only the tumor to identify the presence of DoCM mutations? I was under the impression we only wanted to identify somatic events, in this case DoCM mutations unique to the tumor BAM/CRAM.